.github/workflows/connector-tests.yml

```diff
@@ -74,21 +74,18 @@ jobs:
             cdk_extra: n/a
           - connector: source-chargebee
             cdk_extra: n/a
-          # Currently not passing CI (unrelated)
```

Comment:
This should be fixed now. Why are we removing source-zendesk-support?

Reply:
@maxi297 - Happy to restore it. I'll do that now.

Reply:
Done ✅

Reply:
It seems like `ticket_activities` is missing some records. I don't know if they expire. @DanyloGL would you be able to check this one?